projects/1.16/assets/tinkers-construct/tconstruct/book/tinkers_gadgetry/zh_cn/gadgets/fancy_frame.json

```
@@ -1,13 +1,13 @@
 {
-  "title": "Fancy Item Frames",
+  "title": "花哨的物品展示框",
```

```suggestion
  "title": "精致物品展示框",
```
参考943对fancy的翻译，改为“精致的”。